web/src/components/vis/ScorePlotTile.vue

```diff
@@ -44,6 +48,32 @@ export default {
     pcY() {
       return Number.parseInt(this.pcYval, 10);
     },
+
+    pcCoords() {
+      return this.maybeData('x', []);
+    },
+
+    rowLabels() {
+      return this.maybeData('rows', []);
+    },
+
+    groupLabels() {
+      return this.maybeData('labels', {});
+    },
+
+    eigenvalues() {
+      return this.maybeData('sdev', []);
+    },
+  },
+
+  methods: {
+    maybeData(key, dflt) {
```

Comment:
There's already a method for this on the plotData mixin called `getPlotDataProperty`.  It doesn't do default values but it could be made to.  I think this is generic enough that we can let the mixin do it.

Reply:
I will extend that method to handle default values and replace the invocations to `this.maybeData()` here.

Reply:
See aab86b7fc83ed78e80e122d4b956a573fc59a190. I'm leaving this conversation unresolved in case I did something wrong here.